MetafileImporter/Logic/vtkSlicerMetafileImporterLogic.cxx

```diff
@@ -421,6 +434,85 @@ vtkMRMLSequenceBrowserNode* vtkSlicerMetafileImporterLogic::ReadSequenceMetafile
   return sequenceBrowserNode.GetPointer();
 }
 
+//----------------------------------------------------------------------------
+vtkMRMLSequenceBrowserNode* vtkSlicerMetafileImporterLogic::ReadSequenceNRRD(const std::string& fileName, vtkCollection* addedSequenceNodes/*=NULL*/)
+{
+  // Map the frame numbers to timestamps
+  std::map< int, std::string > frameNumberToIndexValueMap;
+
+  std::string fileNameName = vtksys::SystemTools::GetFilenameName(fileName);
+
+  if (addedSequenceNodes)
+  {
+    addedSequenceNodes->RemoveAllItems();
+  }
+
+#ifdef ENABLE_PERFORMANCE_PROFILING
+  vtkNew<vtkTimerLog> timer;
+  timer->StartTimer();
+#endif
+
+  std::deque< vtkSmartPointer<vtkMRMLSequenceNode> > createdSequenceNodes;
+  std::map< std::string, std::string > imageMetaData;
+  vtkMRMLLinearTransformSequenceStorageNode::ReadSequenceMetafileTransforms(fileName, this->GetMRMLScene(), createdSequenceNodes,
+    frameNumberToIndexValueMap, imageMetaData, true);
```

Comment:
instead of a bool flag, use an enum like NRRD_SEQUENCE_FILE, METAIMAGE_SEQUENCE_FILE (it makes the code easier to read)